src/lib/api/supabase/projects/api.ts

```diff
@@ -39,6 +64,7 @@ export async function writeDBProjects(records: Json<DBProject>[]) {
 
   const queue = records.map(r => ({
     ...formatDBProjectRow(r),
+    archived: r.archived || isHardArchived(r), // store a project's hard archived state in the db
```

Comment:
we'll now store a project's archived state in the DB, _including_ if it has been manually archived

Reply:
nice